src/Template/Bake/Template/view.ctp

```diff
@@ -49,23 +49,33 @@ $groupedFields = collection($fields)
     ->toArray();
 
 $groupedFields += ['number' => [], 'string' => [], 'boolean' => [], 'date' => [], 'text' => []];
-$pk = "\$$singularVar->{$primaryKey[0]}";
+$primaryKeyArguments = [];
+foreach ($primaryKey as $primaryKeyComponent) {
+    $primaryKeyArguments[] = '$' . $singularVar . '->' . $primaryKeyComponent;
+}
+$primaryKeyArgumentList = join($primaryKeyArguments, ', ');
```

Comment:
It exists twice. Here and in form.ctp. I'd not know where to add the helper method.